src/main/resources/templates/springboot/rest/build.gradle.tmpl

```diff
@@ -32,12 +32,18 @@ dependencies {
 
     clientCompile "com.blackbaud:common-spring-boot-rest:\${springBootVersion}-2.+"
 
+    mainTestCompile "com.blackbaud:common-spring-boot-rest-test:\${springBootVersion}-2.+"
     mainTestCompile "com.blackbaud:common-test:1.+"
     mainTestCompile "org.codehaus.groovy:groovy-all:2.4.3"
     mainTestCompile ("org.spockframework:spock-spring:1.0-groovy-2.4") {
         exclude group: "org.codehaus.groovy"
     }
 
+    testCompile ('org.spockframework:spock-spring:1.0-groovy-2.4') {
+        exclude group: 'org.codehaus.groovy'
+    }
+    testCompile "org.springframework.boot:spring-boot-starter-test"
+
     componentTestCompile "org.springframework.boot:spring-boot-starter-test"
```

Comment:
just change these two to be mainTestCompile and they'll be pulled in for both unit and component tests